cirq-core/cirq/_compat.py

```diff
@@ -15,6 +15,8 @@
 """Workarounds for compatibility issues between versions and libraries."""
 import contextlib
 import dataclasses
+
```

Comment:
nit: remove blank line and sort imports since `contextvars` is part of the stdlib.